### ISSUE TITLE
remote participant audio quality is often poor

### ISSUE THREAD

**Issue by naikrovek** [thechangelog/changelog.com#362]
I have listened to a few recent Go Time podcasts, and I noticed that remote participants seem to have poor quality (meaning low bitrate) audio quality.

Can you, if you haven't yet, try something like [SonoBus](https://sonobus.net/) ([GitHub link](https://github.com/sonosaurus/sonobus)) for remote participants?  It is designed to do what you need; high quality, low latency audio over the internet.

Obviously this is more of a suggestion than an issue; I can hear and understand remote participants just fine.  I do not need to be informed about any decisions to use this tool or any other, I just wanted to make people aware of it.

**Comment by jerodsanto** [comment]
Thanks for making us aware of this! We've been searching for a solution in this space and have never heard of SonoBus. I'll check it out 💯

**Comment by naikrovek** [comment]
Did anything ever come of this?  Was that software useful?  I can look for more if it isn't sufficient in some way.  It's not exactly easy to use, that software.

**Comment by jerodsanto** [comment]
Yeah it was just too complex to ask our guests to use it. 

We record 6(ish) shows a week, which is a _lot_ of guest relations, so our goal is to reduce recording complexity. To that end, we've adopted [Riverside](https://riverside.fm) as our platform for now. It captures better local audio for each participant than we used to get with self-recording (for various human error reasons), streams video to YouTube for our live shows, and doesn't require _any_ local installation if you already have a Chromium-based browser (which most people do).

I'd love to have _excellent_ guest audio for every episode, but at this point it's more about their recording environment, microphone, etc, than it is latency and bitrate.
